config/environment.rb

```diff
@@ -1,5 +1,7 @@
 # Load the Rails application.
 require_relative "application"
+require "dotenv"
+Dotenv.load
```

Comment:
can you please tell the purpose of DotEnv??

Reply:
bhaiya dotenv is used for hiding the sensitive data like different secret keys, credentials etc